### PR TITLE
update forc and fuels-rs

### LINKED ISSUE #16: Update to latest forc and sdk versions

### PR DESCRIPTION
**Pull request by pixelcircuits**

### CODE REVIEW

File: contract-message-script/src/transaction_utils.sw [hunk 1]
```diff
@@ -1,8 +1,9 @@
 library transaction_utils;
 
+use std::inputs::GTF_INPUT_MESSAGE_AMOUNT;
```
Comment (nfurfaro) [inline contract-message-script/src/transaction_utils.sw:3]: Hmm. the spending script uses this file, and this file has changed. I feel like the script hash should have also changed as a result, but the hardcoded script hash in the predicate is still the same. What am I missing?

Comment (pixelcircuits) [reply contract-message-script/src/transaction_utils.sw:3]: The value for `GTF_INPUT_MESSAGE_AMOUNT` and `GTF_INPUT_CONTRACT_CONTRACT_ID` didn't change. We're just referencing them from the std lib now

Comment (nfurfaro) [reply contract-message-script/src/transaction_utils.sw:3]: yeah makes sense.

**Review by nfurfaro — APPROVED** [review]
Looks good.

**Pull request merged**